src/java.base/share/classes/java/math/BigDecimal.java

```diff
@@ -79,17 +78,23 @@
  * {@code BigDecimal} arithmetic are broadly compatible with selected
  * modes of operation of the arithmetic defined in ANSI X3.274-1996
  * and ANSI X3.274-1996/AM 1-2000 (section 7.4).  Unlike those
- * standards, {@code BigDecimal} includes many rounding modes, which
- * were mandatory for division in {@code BigDecimal} releases prior
- * to 5.  Any conflicts between these ANSI standards and the
+ * standards, {@code BigDecimal} includes many rounding modes
+ * Any conflicts between these ANSI standards and the
  * {@code BigDecimal} specification are resolved in favor of
  * {@code BigDecimal}.
  *
  * <p>Since the same numerical value can have different
  * representations (with different scales), the rules of arithmetic
  * and rounding must specify both the numerical result and the scale
  * used in the result's representation.
- *
+
+ * The different representations of same numerical value are called
```

Comment:
Should that be:
The different representations of *a* same numerical value are called ...  - or maybe value is missing a 's'?

Reply:
Missing "the" -- will fix; thanks.